analytics/terraform/datahub-on-eks/helm-values/kubecost-values.yaml

```diff
@@ -0,0 +1,104 @@
+
+# KubeCost WebUI -> kubectl port-forward --namespace kubecost deployment/kubecost-cost-analyzer 9090
+nodeSelector:
+  NodeGroupType: core
+# --set nodeSelector."beta\\.kubernetes\\.io/arch"=arm64
+
+priority:
+  enabled: false
+
+networkPolicy:
+  enabled: false
+
+podSecurityPolicy:
+  enabled: false
+
+global:
+  grafana:
+    enabled: false
+    proxy: false
+
+  # This Prometheus setup is reusing the existing Prometheus deployment
+  # Check for more docs under https://guide.kubecost.com/hc/en-us/articles/4407595941015
+  prometheus:
+    fqdn: http://prometheus-server.prometheus.svc
```

Comment:
Does this work with Kube Prometheus Stack addon? Have you verified KubeCost from WebUI? You can remove this add-on if its not important for Datahub blueprint

Reply:
kubecost addon had been removed before.  removed the helm value file now